MathMountain/BlockView.swift

```diff
@@ -19,7 +19,14 @@ struct BlockView: View {
             .padding()
             .background(.gray)
             .cornerRadius(8)
+            .opacity(isDestroyed ?
+                     0.0 : 1.0)
             .frame(minWidth: UIScreen.main.bounds.width * 0.125)
+            .onTapGesture {
+                if number == answer {
+                    isDestroyed = true
```

Comment:
the designers asked for an animation when the block is destroyed, could you add that?

Reply:
Hey added an animation, let me know if it is okay!